### DISCUSSION TITLE
"Source Fade" default setting toggle ON/OFF

### DISCUSSION THREAD

**Discussion by FakenameMusic** [helgoboss/helgobox#1151] [Ideas]
For users like me who prefer to manage their own fades, or others who will only work with premade loops, it would be good to have the option of "always off" for the source fade.

Source discussion: https://github.com/helgoboss/helgobox/discussions/1142#discussioncomment-10499710

**Comment by helgoboss** [comment]
Could you link to that other discussion (don't have it ready right now)?

**Reply by FakenameMusic** [reply]
Done!

**Reply by helgoboss** [reply]
Thanks!

Assuming I implemented things like this: https://github.com/helgoboss/helgobox/discussions/1142#discussioncomment-10490358 ...


... Do you think this feature is really necessary? Do you really want to have source fades turned off even for recordings within Playtime and for item imports that have fades (which would be exactly the consequence of your feature request)?

**Reply by FakenameMusic** [reply]
Ah! I have just re-read your points carefully and it appears that "If item doesn't have a fade at its tart, disable "Source fades" (this in particular would have prevented your issue)."  Would cover this completely...

I will close this thread!
